README.md

```diff
@@ -3,88 +3,6 @@
 </div>
 
 -----------------
+This forked tensorflow repository is used for [Parallax](https://github.com/snuspl/parallax). It changes original tensorflow-v1.6 to keep all the gradients and its target tensors made by automatic differentiatiion in [TensorFlow collection](https://www.tensorflow.org/api_guides/python/framework#Graph_collections). We also add `average_option` in `SparseConditionalAccumulator` to support the diverse average patterns of language models(`average_option` : Integer indicating how to average accumulated values(1: divide by the accumulation count, 2: divide by each element counter, 3: no division).
```

Comment:
_the_ original tensorflow-v1.6
differentiation
average out the accumulated values